packages/core/admin/server/controllers/admin.js

```diff
@@ -26,11 +26,24 @@ module.exports = {
   // When removing this we need to update the /admin/src/index.js file
   // where we set the strapi.window.isEE value
   async getProjectType() {
+    const promoteEnterpriseFeatures = strapi.config.get('admin.promoteEnterpriseFeatures', true);
     // FIXME
     try {
-      return { data: { isEE: strapi.EE, features: ee.features.list() } };
+      return {
+        data: {
+          isEE: strapi.EE,
+          features: ee.features.list(),
+          promoteEnterpriseFeatures,
```

Comment:
If we want to move this forward, I would say this would be a great place to place this.

I would not promote enterprise features if EE is enabled  though
```suggestion
          strapi.EE ? false : promoteEnterpriseFeatures,
```